src/Maestro/Maestro.Web/Pages/Account/AccountController.cs

```diff
@@ -14,18 +15,22 @@ public class AccountController : Controller
 {
     [HttpGet("/Account/SignOut")]
     [AllowAnonymous]
-    public new async Task<IActionResult> SignOut()
+    public new async Task SignOut()
     {
-        await HttpContext.SignOutAsync();
-        return Redirect($"{Request.Scheme}://{Request.Host}");
```

Comment:
This redirect is why the "Sign Out" link currently does nothing other than redirect to the home page on https://maestro.dot.net/. The OIDC handler's signout functionality relies on setting a `Location` header to redirect to Entra, but it gets clobbered.

It also didn't previously clear the authentication cookies since `SignOutAsync()` only signs out of the default scheme which is OIDC.

Reply:
Thanks for fixing this